yaksh/models.py

```diff
@@ -1181,7 +1193,7 @@ def can_attempt_now(self, user, course_id):
                 user=user, questionpaper=self, course_id=course_id
             )
             if last_attempt:
-                time_lag = (timezone.now() - last_attempt.start_time).days
+                time_lag = (timezone.now() - last_attempt.start_time).total_seconds() / 3600
```

Comment:
I will try and change the message.

P.S. This message was not a part of my changeset

Reply:
Agreed this wasn't a part of your change set. But if it is wrong, someone has to change it. And since your PR does come close, hence the aforementioned comment.